### PR TITLE
drivers: can: shell: add support for setting SJW

### PR DESCRIPTION
**Pull request by henrikbrixandersen**
Add support for setting the Synchronization Jump Width (SJW) for both the classic/arbitration phase and the CAN-FD data phase.

### CODE REVIEW

File: drivers/can/can_shell.c [hunk 1]
```diff
@@ -366,7 +366,15 @@ static int cmd_can_bitrate_set(const struct shell *sh, size_t argc, char **argv)
 			return -EINVAL;
 		}
 
-		timing.sjw = CAN_SJW_NO_CHANGE;
+		if (argc >= 5) {
+			timing.sjw = (uint16_t)strtoul(argv[4], &endptr, 10);
```
Comment (bbilas) [inline drivers/can/can_shell.c:370]: Why not dedicated `shell_strtoul`?

Comment (henrikbrixandersen) [reply drivers/can/can_shell.c:370]: Sure, but the rest of the file uses `strtoul()`. I must admit I don't see much benefit from using `shell_strtoul()` over `strtoul()` here.

Comment (henrikbrixandersen) [reply drivers/can/can_shell.c:370]: @bbilas I will follow up with a PR for converting the entire file to use `shell_strtoul()`. Okay?

Comment (bbilas) [reply drivers/can/can_shell.c:370]: Sure, I was just curious ;) I have no strong objections.

**Review by bbilas — APPROVED** [review]

**Review by str4t0m — APPROVED** [review]

**Pull request merged**